.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.4']
+        wordpress: ['4.7', 'latest-stable', 'latest-stable (multisite)', 'nightly']
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: wordpress
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          extensions: mbstring, intl, redis, pdo_mysql
+          php-version: ${{ matrix.php }}
+      - name: Start MySQL
+        run: sudo service mysql start
+      - name: Enable multisite
+        if: ${{ matrix.wordpress == 'latest-stable-multisite' }}
+        run: |
+          echo "WP_MULTISITE=1" >> $GITHUB_ENV
```

Comment:
As mentioned before, this can be set unconditionally in the job's `env` (or in the test step) based on the value in the matrix. By default we can set it to `0` if needed, but probably unset/blank is fine too since this needs to be truthy to enable multisite.